documentation/release_5.2.htm

```diff
@@ -26,6 +26,10 @@ <h2> Summary for end users (also to be read by developers)</h2>
 
 <h3>Bug fixes</h3>
 <ul>
+  <li>Scatter estimation was setting initial activity image to 1 at set_up and during each iteration. This led to cases where the scatter estimation would not converge.
+    <br /><a href="https://github.com/UCL/STIR/pull/1160/">PR #1160</a>. Now, this is disabled by default, but can be re-enabled by setting <code>restart_reconstruction_every_scatter_iteration</code>
+    to true either via a parameter file or via the <code>set_restart_reconstruction_every_scatter_iteration()</code> function.
+  </li>
```

Comment:
I wouldn't call this a bug fix really. Also, it makes the results of the scatter backwards incompatible so this needs a fairly big warning. Maybe I'll try to edit this later...